catalog-rest-service/src/main/resources/json/schema/entity/services/ingestionPipelines/ingestionPipeline.json

```diff
@@ -164,7 +164,8 @@
     },
     "loggerLevel": {
       "description": "Set the logging level for the workflow.",
-      "$ref": "../../../metadataIngestion/workflow.json#/definitions/logLevels"
+      "$ref": "../../../metadataIngestion/workflow.json#/definitions/logLevels",
+      "default": "INFO"
```

Comment:
python generation does not handle this very well. It adds the `default` as `LogLevels.INFO` without importing this first, which breaks the REST plugins. I'll remove this line and see

Reply:
@pmbrull, JsonSchema2Pojo does not set the default value, if it is set when the type is defined and not at the place where $ref is used to reference it.